models/fixtures/repo_unit.yml

```diff
@@ -550,3 +550,24 @@
   repo_id: 53
   type: 1
   created_unix: 946684810
+
+-
+  id: 81
+  repo_id: 54
+  type: 4
+  config: "{}"
+  created_unix: 946684810
+
+-
+  id: 82
+  repo_id: 54
+  type: 5
+  config: "{}"
+  created_unix: 946684810
```

Comment:
Do I really need all of these? I see in 36a2d2f919ee445f0374c210309b336e7c3c1f8e that only one permission needed to be added for a new repo.

```suggestion
```

Reply:
(I removed the extra two)